modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMAnnouncements.macro

```diff
@@ -0,0 +1,18 @@
+definition {
+
+	@summary = "Add Announcement and Publish in page the PRM"
```

Comment:
Hi @marinaasant0s, could you please rename this summary to: "Create and Publish an Announcement"